src/bidiMapper/domains/network/NetworkProcessor.ts

```diff
@@ -334,6 +338,17 @@ export class NetworkProcessor {
           return urlPattern;
         }
         case 'pattern':
+          // No params signifies intercept all
+          if (
```

Comment:
Is this chunk really needed, given the change to return `'*'` in `buildUrlPatternString`?